src/main/java/randoop/resource/MethodListGen.java

```diff
@@ -0,0 +1,241 @@
+package randoop.resource;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static randoop.reflection.OperationModel.signatureToOperation;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+import org.plumelib.reflection.Signatures;
+import randoop.main.RandoopUsageError;
+import randoop.reflection.EverythingAllowedPredicate;
+import randoop.reflection.VisibilityPredicate;
+import scenelib.annotations.Annotation;
+import scenelib.annotations.el.AClass;
+import scenelib.annotations.el.AMethod;
+import scenelib.annotations.el.AScene;
+import scenelib.annotations.io.classfile.ClassFileReader;
+
+/**
+ * This tool generates the resource files JDK-sef-methods.txt and omitmethods-defaults.txt, which
+ * contain a list of side effect free methods and a list of methods to omit during Randoop
+ * execution, respectively.
+ */
+public class MethodListGen {
+  /**
+   * The extension for a Java .class file. These files contain annotations that this tool will
+   * parse.
+   */
+  private static final String CLASS_EXT = ".class";
```

Comment:
This doesn't serve any purpose.  It will never change, and it is used only once.  Use the string literal there and you can delete lines 38-43.